### DISCUSSION TITLE
Help on hmUI.system_status.LOCK datatype

### DISCUSSION THREAD

**Discussion by dave-chk** [zepp-health/zeppos-forum#56] [Q&A] [answered]
Hi, I'm trying to create a watchface for a miband 7 which relies on zepp os apparently. 
I see that I can have an IMG_STATUS type widget that leverages on the hmUI.system_status.LOCK datatype to show if the screen lock is active or not.
It works but not very good:  it is show based on the option of the screen lock, not the actual status of the screen:
If I don't enable screen lock in settings, then it is not shown, if I enable it it is shown, but it remains shown even after I've unlocked the screen.

Is there a datatype or a different way to show the actual status? 
The result I would like to have is that if the option is on but currently the screen is unlocked then the icon should not be shown.

Thanks a lot.

**Comment by MyoungXUE** [comment]
Hello, sorry for the long wait due to the Chinese New Year.

I have passed this question to the relevant colleagues in charge, please wait patiently for our reply again.

**Comment by MyoungXUE** [comment, marked as answer]
I confirmed with my colleague that `hmUI.system_status.LOCK` only represents the status of the lock screen setting switch, not the lock status of the screen.

Unfortunately, there is no way to get the lock status of the screen.

**Reply by dave-chk** [reply]
Thanks a lot for confirming this, I'm sorry that there is no way but at least I know 😊
